samcli/commands/_utils/options.py

```diff
@@ -1009,3 +1009,22 @@ def watch_exclude_click_option():
         type=SyncWatchExcludeType(),
         callback=watch_exclude_option_callback,
     )
+
+
+def container_env_var_file_click_option(cls):
+    """
+    Click option to --container-env-var-file option
+    """
+    return click.option(
+        "--container-env-var-file",
+        "-ef",
+        default=None,
+        type=click.Path(),  # Must be a json file
+        help="Environment variables json file (e.g., env_vars.json) to be passed to containers.",
+        cls=cls,
+    )
+
+
+@parameterized_option
+def container_env_var_file_option(f, cls):
+    return container_env_var_file_click_option(cls)(f)
```

Comment:
Sorry to bug you again, but we don't need to keep `ContainerOptions` as a parameter since it will be the case all the time. Can we directly use the type in the newly created function above?

I am adding my suggestion in the comments, let me know if it makes sense to you so that we can apply it quickly (we might need to add the import the `ContainerOptions` class in this file though).
```suggestion
def container_env_var_file_click_option():
    """
    Click option to --container-env-var-file option
    """
    return click.option(
        "--container-env-var-file",
        "-ef",
        default=None,
        type=click.Path(),  # Must be a json file
        help="Environment variables json file (e.g., env_vars.json) to be passed to containers.",
        cls=ContainerOptions,
    )


@parameterized_option
def container_env_var_file_option(f):
    return container_env_var_file_click_option()(f)
```

Reply:
@mndeveci makes sense. So i had one doubt should i move `ContainerOptions` class to some common place like inside `_utils` module ?

Reply:
@nitintecg I've checked other places and it looks like this is a pattern that we use before. So I will keep as it is now since this will touch other places.